### PR TITLE
 Bug 1561617 - Use mozilla-version to ensure new Fennec versions are supported 

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: treescript/versionmanip.py [hunk 1]
```diff
@@ -54,43 +55,46 @@ def _get_version(file):
 
     """
     bump_info = get_version_bump_info(context.task)
-    next_version = bump_info['next_version']
-    old_next_version = None
     files = bump_info['files']
     changed = False
+
+    VersionClass = _find_what_version_parser_to_use(files)
```
Comment (Callek) [inline treescript/versionmanip.py]: I'd rather we check the parser based on file to bump - specifically milestone.txt is a gecko version and I can envision it being bumped with different products in the future, should we get there... thoughts?

Comment (JohanLorenzo) [reply treescript/versionmanip.py]: Sure, I can use `GeckoVersion` for `milestones.txt`! Good idea!

Comment (JohanLorenzo) [reply treescript/versionmanip.py]: Done!

**Comment by JohanLorenzo** [comment]
Next week is the release candidate week. This patch isn't needed by then. I'll land/deploy it after 68.0 is out.

**Comment by JohanLorenzo** [comment]
We bump Fennec beta to 68.1b1 tomorrow. I'm going to deploy these changes

**Pull request merged**